### PR TITLE
Add dogfood builds.

### LINKED ISSUE #94: Add dogfood variant
Include Telescope + first-run tutorial.

### PR DESCRIPTION
**Pull request by JakeWharton**
This is accomplished by splitting the APK into 'production' and 'internal' flavors. The debug variant of 'internal' has all the fancy drawer things. The release variant of 'internal' only has Telescope for bug reporting. The 'production' flavor (both debug and release) are stripped of all this hotness–providing debuggable release builds (b-b-b-bonus!).

Closes #94.

### CODE REVIEW

File: src/internalRelease/java/com/jakewharton/u2020/ui/TelescopeAppContainer.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import butterknife.InjectView;
+import com.jakewharton.u2020.R;
+import com.jakewharton.u2020.data.LumberYard;
+import com.jakewharton.u2020.ui.bugreport.BugReportLens;
+import com.mattprecious.telescope.TelescopeLayout;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+@Singleton
+public final class TelescopeAppContainer implements AppContainer {
```
Comment (JakeWharton) [inline src/internalRelease/java/com/jakewharton/u2020/ui/TelescopeAppContainer.java]: Eventually I think we'll kill this, move `DebugAppContainer` into `src/internal/`, and make its contents modular.

**Comment by JakeWharton** [comment]
RIP Doge images :cry: 

SUCH U+2020

File: src/internalRelease/java/com/jakewharton/u2020/ui/TelescopeAppContainer.java (hunk 1 above)
Comment (mattprecious) [inline src/internalRelease/java/com/jakewharton/u2020/ui/TelescopeAppContainer.java]: :+1:

**Comment by mattprecious** [comment]
This was a lot simpler than I was expecting. LG!

**Pull request merged**